### PR TITLE
Figured out local CMS development, removed unused / unneeded files

### LINKED ISSUE #26: Get CMS working from local build
This starter / example had the ability to run the Netlify CMS from a local build ... which was very handy.
https://github.com/AustinGreen/gatsby-starter-netlify-cms

I haven't got that working yet, but it would be nice to have / document.

### PR DESCRIPTION
**Pull request by product-mindset**
Fixes #26 by adding to `README`.  Will create follow-up ticket to address automatically. Removes several unused and uneeded files.  In general these were for the CMS. Plugins can generate them automatically until customization is needed.

### CODE REVIEW

File: static/admin/index.html [hunk 1]
```diff
@@ -1,16 +0,0 @@
-<!doctype html>
```
Comment (product-mindset) [inline static/admin/index.html]: Automatically generated by `gatsby-plugin-netlify-cms`.

**Comment by derektiffany** [comment]
Deploy preview for *product-mindset* ready!

Built with commit d0707c4a0d37007a974887fe88f4eb9a209fe0c7

https://deploy-preview-45--product-mindset.netlify.com

**Comment by product-mindset** [comment]
`cms.css` is only needed if we're generating custom previews (which may be good eventually, but currently not needed and is just wasting space in the code base)

File: src/cms/cms.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// tslint:disable-next-line:no-implicit-dependencies
-import CMS from 'netlify-cms'
-
-CMS.registerPreviewStyle('/cms.css')
+// Your module must at least include these three imports
+import React from "react";
+import CMS from "netlify-cms";
+import "netlify-cms/dist/cms.css";
```
Comment (product-mindset) [inline src/cms/cms.js]: cms.css is only needed if we're generating custom previews (which may be good eventually, but currently not needed and is just wasting space in the code base)

**Pull request merged**